.babelrc

```diff
@@ -0,0 +1,3 @@
+{
+  "presets": ["react", "es2015", "stage-0"]
```

Comment:
I don't think we want to support stage-0. Its likely to change and we don't really _need_ many of the features it uses (stuff like generators and await will likely confuse a lot of folks)

I do think it would be nice to have decorators though